developers/weaviate/current/configuration/authentication.md

```diff
@@ -116,29 +116,45 @@ services:
       AUTHENTICATION_OIDC_SKIP_CLIENT_ID_CHECK: 'false'
 ```
```

Comment:
Not entirely sure, but should we guide users which values to enter here? That was one of the things that took me forever to figure out

Reply:
Am I right that our current flows and providers work without modifying this parameter? 
If that is the case, I wonder if we can leave this as is. Please feel free to suggest any changes of course.

Reply:
yes, I haven't touched this parameter